hack/functest.sh

```diff
@@ -1,3 +1,8 @@
-#!/bin/bash -e
+#!/usr/bin/env bash
 
-echo hello world
+set -xe
+
+source ./cluster/cluster.sh
+
+export KUBEVIRT_CLIENT_GO_SCHEME_REGISTRATION_VERSION=v1
+KUBECONFIG=${KUBECONFIG:-$(cluster::kubeconfig)} $GO test -test.timeout=1h -race -test.v ./tests/... $E2E_TEST_ARGS -ginkgo.v
```

Comment:
please add deterministic seed of 0 (unless we do want tests to run on random order once we have more)
on kubevirt we use fixed seed

Reply:
Currently we have just one test, so it is not relevant.
Anyway, I think it is healthier to run the tests in a random order.